content/sdk/10.js/00.ethers/10.v5/10.types.md

```diff
@@ -1,23 +1,36 @@
 ---
 title: Types and Interfaces
-description:
+description: Definitions and interfaces used in ZKsync Era.
+tags: ["ZKsync", "types", "interfaces", "Ethereum", "blockchain"]
 ---
 
 ## `AccountAbstractionVersion`
 
 Enumerated list of account abstraction versions.
 
-- None = `0` - Used for contracts that are not accounts.
-- Version1 = `1` - Used for contracts that are accounts.
+::collapsible
+  ::field{name="None" type="number" defaultValue="0"}
+  Used for contracts that are not accounts.
+  ::
+  ::field{name="Version1" type="number" defaultValue="1"}
+  Used for contracts that are accounts.
+  ::
+::
```

Comment:
I don't believe the AccountAbstractionVersion and AccountNonceOrdering values are properties. Those are actual values which you have to use to indicate either None or Version1. The `defaultValue` isn't even a valid property on the `field` component. I think you want to use `default`, but once again this is not an appropriate use of the properties UI for these values.